### PR TITLE
update users management for enterprise

### PR DESCRIPTION
**Pull request by ayatsyny**

### CODE REVIEW

File: crowdin_api/api_resources/users/resource.py [hunk 1]
```diff
@@ -44,6 +39,45 @@ def list_project_members(
...
+
+class UsersResource(BaseUsersResource):
+    """
+    Resource for Users.
+
+    Users API gives you the possibility to get profile information about the currently
+    authenticated user.
+
+    Link to documentation:
+    https://support.crowdin.com/api/v2/#tag/Users
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/users/resource.py:51]: Wrong link

File: crowdin_api/api_resources/users/resource.py [hunk 2]
```diff
@@ -44,6 +39,45 @@ def list_project_members(
...
+    """
+
+    def get_member_info(self, projectId: int, memberId: int):
+        """
+        Get Member Info.
+
+        Link to documentation:
+        https://support.crowdin.com/api/v2/#operation/api.projects.members.get
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/users/resource.py:59]: Wrong link

File: crowdin_api/api_resources/users/resource.py (hunk 1 above)
Comment (ayatsyny) [reply crowdin_api/api_resources/users/resource.py]: fixed

File: crowdin_api/api_resources/users/resource.py (hunk 2 above)
Comment (ayatsyny) [reply crowdin_api/api_resources/users/resource.py]: fixed

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**